### DISCUSSION TITLE
homebrew batch file?

### DISCUSSION THREAD

**Discussion by Longshoez** [Homebrew/discussions#1881] [Everyday usage]
I downloaded homebrew because i wanted to install ppsspp on my mac, but it's a pain in the ass to open the terminal and type ppsspp just to open the app, is there a way i can create a shortcut or something? im looking for something that works like a batch file in windows.

**Comment by carlocab** [comment]
You're probably looking for something like this: https://mathiasbynens.be/notes/shell-script-mac-apps

There are many alternative ways to do this, so you may want to do a bit of research, but I remember using the method above and it working out for me.
